src/components/annotation-form.jsx

```diff
@@ -17,79 +18,101 @@ import {Version} from '../services/web-monitoring-db';
-                    <Checkbox {...common} name="repeat_10">Sidebar removal or change</Checkbox>
-                    <Checkbox {...common} name="repeat_11">Banner/advertisement removal or change</Checkbox>
-                    <Checkbox {...common} name="repeat_12">Scrolling news/reports</Checkbox>
-                </div>
-                <div className="col-md-4">
-                    <h4>Significance</h4>
-                    <Checkbox {...common} name="sig_1">Change related to energy, environment, or climate</Checkbox>
-                    <Checkbox {...common} name="sig_2">Language is significantly altered</Checkbox>
-                    <Checkbox {...common} name="sig_3">Content is removed</Checkbox>
-                    <Checkbox {...common} name="sig_4">Page is removed</Checkbox>
-                    <Checkbox {...common} name="sig_5">Insignificant</Checkbox>
-                    <Checkbox {...common} name="sig_6">Repeated Insignificant</Checkbox>
-                </div>
-            </form>
-        );
-    }
+        <textarea
+          name="further-notes"
+          placeholder="Further notes"
+          onChange={this._onNotesChange}
+          value={annotation.notes || ''}
+        />
+      </form>
+    );
+  }
 
-    _onFieldChange (valueObject) {
-        if (this.props.onChange) {
-            const newAnnotation = Object.assign({}, this.props.annotation, valueObject);
-            this.props.onChange(newAnnotation);
-        }
+  _onFieldChange(valueObject) {
+    if (this.props.onChange) {
+      const newAnnotation = Object.assign({}, this.props.annotation, valueObject);
+      this.props.onChange(newAnnotation);
     }
+  }
+
+  _onNotesChange(event) {
+    this._onFieldChange({ notes: event.target.value });
+  }
 }
 
 AnnotationForm.defaultProps = {
-    annotation: null,
-    collapsed: true
+  annotation: null,
+  collapsed: true
 };
 
-function Checkbox ({children, formValues, name, onChange}) {
-    const fieldNumber = name.split('_')[1];
-    const checked = !!(formValues && formValues[name]);
-    const changeHandler = (event) =>
-        onChange({[name]: event.currentTarget.checked});
-
-    return (
-        <label>
-            <input type="checkbox" name={name} checked={checked} onChange={changeHandler} />
-            {fieldNumber}
-            <span className="info-text">= {children}</span>
-        </label>
-    );
+function Checkbox({ children, formValues, name, onChange }) {
+  const fieldNumber = name.split('_')[1];
+  const checked = !!(formValues && formValues[name]);
+  const changeHandler = (event) =>
+    onChange({ [name]: event.currentTarget.checked });
+  return (
+    <li className="signifier-list-item">
+      <input type="checkbox" className="toggle" id={name} checked={checked} onChange={changeHandler} />
+      <label data-tip={children} data-for={`${name}-tip`} className="toggle-btn" data-tg-on={fieldNumber} htmlFor={name} />
+      <span className="info-text">{children}</span>
```

Comment:
I was able to insert the span into the label so that it would all be clickable. Also, realized I could show or hide it using conditional rendering rather than css. Starting to get it, React rocks!

Still had to keep the `input` and `label` separate because of the unique styling. The links you recommended all do the same as well. I added a unique ID for the fields